dist/bearhug-angular.js

```diff
@@ -0,0 +1,207 @@
+(function(angular) {
```

Comment:
This is generated output, but committed to the repo because it'll be needed for bower installation.